velox/exec/WindowFunction.h

```diff
@@ -111,10 +111,12 @@ class WindowFunction {
       const core::QueryConfig& config);
 
  protected:
-  // Set result NULL for rows with invalid frames in the input.
-  void setNullEmptyFramesResults(
+  // Set result to NULL for rows with invalid frames in the input when
```

Comment:
We are always providing a defaultResult right ? So this explanation is incorrect. 

It might be enough to say
"Set result to default value for rows with empty frames."

Reply:
Thanks for pointing it out, updated.